app/src/main/java/ru/anisart/vv/MainActivity.kt

```diff
@@ -33,17 +33,17 @@ import java.util.*
 @RuntimePermissions
```

Comment:
**Reporter**: ANDROIDLINT
**Rule**: SetJavaScriptEnabled
**Severity**: WARN
**File**: C:/Users/aanisimov/.jenkins/workspace/vv/app/src/main/java/ru/anisart/vv/MainActivity.kt L64

Using &#96;setJavaScriptEnabled&#96;&#10;Using &#96;setJavaScriptEnabled&#96; can introduce XSS vulnerabilities into your application, review carefully.&#10;Your code should not invoke &#96;setJavaScriptEnabled&#96; if you are not sure that your app really requires JavaScript support.


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a882791664>*